x-pack/solutions/security/plugins/security_solution/public/detections/pages/detection_engine/rules/helpers.tsx

```diff
@@ -230,7 +232,11 @@ export const getHumanizedDuration = (from: string, interval: string): string =>
   return secondsToDurationString(intervalDuration);
 };
 
-export const getAboutStepsData = (rule: RuleResponse, detailsView: boolean): AboutStepRule => {
+export const getAboutStepsData = (
+  rule: RuleResponse,
+  detailsView: boolean,
+  euiTheme: EuiThemeComputed
```

Comment:
`euiTheme` variable is not used anywhere.
It forces also to pass `euiTheme` to parent functions as well